dist/sizzle.js

```diff
@@ -917,6 +934,10 @@ Sizzle.attr = function( elem, name ) {
 				null;
 };
 
+Sizzle.escape = function( sel ) {
+	return (sel + "").replace( rcssescape, fcssescape );
```

Comment:
> `CSS.escape` is supported in Chrome and FF, can we degrade instead?

Opted against this because we need our own implementation anyway.

> We accept `null` input only for internal purposes right? Can we expose only standardized part of the API and deal with our syntax internally? Since there might be a reason for the "\0" exceptions that we don't know of.

Handled by a spec change to accept U+0000 NULL and translate to U+FFFD REPLACEMENT CHARACTER.

> Are we planning to expose this through jQuery API too? I would suggest to do it.

Yes; https://github.com/jquery/jquery/issues/1761 .